pkg/sink/sink.go

```diff
@@ -56,6 +55,11 @@ import (
 	v1 "knative.dev/pkg/apis/duck/v1"
 )
 
+const (
+	// EventProcessed is sent for CloudEvent compliant providers
+	EventProcessed = "dev.tekton.event.triggers.processed.v1"
```

Comment:
Also, EventAccepted might be a better name since when the response is sent, the event may not have been fully processed yet as the triggers may still be processing them

Reply:
Done. I didn't include it pkg/reconciler earlier because those are k8s/cloudEvent used during processing. But I think it doesn't matter, it can cater to any events we are using in Triggers.